### PR TITLE
Fix tween update

### PR DESCRIPTION
**Pull request by jojorne**
This fixes the animation on the update icon and any animation that uses the tween method.

### CODE REVIEW

**Comment by jojorne** [comment]
Uh... I think there's something wrong with GitHub's indentation...

**Comment by jojorne** [comment]
Also, I noticed that the animation is being affected by game time. 60 FPS is twice as fast as 30 FPS.

File: Blish HUD/Library/Glide/Tween.cs [hunk 1]
```diff
@@ -55,348 +55,349 @@ public enum RotationUnit
...
-						
-						if (repeatCount > 0) {
+                float t;
+                time += elapsed;
+                if (time >= Duration)
+                {
+                    t = 1;
+                    if (repeatCount != 0)
+                    {
+                        time = 0;
```
Comment (jojorne) [inline Blish HUD/Library/Glide/Tween.cs]: When reseting time, the right way should be:
```csharp
time -= Duration;
```
By doing it this way, you don't lose time precision.

**Comment by entrhopi** [comment]
This is working nicely now. I can't see any issues. Even the rubberbanding effect when framerate is set to unlimited seems to be gone.

**Comment by jojorne** [comment]
Fixed a rare case where opening/closing a window would cause it to blink.

**Comment by dlamkins** [comment]
FYI, I know that the original files were poorly formatted when we pulled and committed them from their original repo, but the formatting changes included in this PR are making it quite hard to tell what specifically was changed for the review.

**Comment by jojorne** [comment]
image
Does this help? Because Visual Studio kept changing the format to follow `.editorconfig` then I manually kept changing it back every time. I was just losing time with the formatting that in the and I gave up... lol

**Review by dlamkins — APPROVED** [review]

**Pull request merged**